src/xdocs/checks/javadoc/atclauseorder.xml

```diff
@@ -88,34 +88,36 @@
         </p>
         <source>
 /**
-* Some javadoc. // OK
+* Some javadoc.
 *
-* @author Some javadoc. // OK
-* @version Some javadoc. // OK
-* @param Some javadoc. // OK
-* @return Some javadoc. // OK
-* @throws Some javadoc. // OK
-* @exception Some javadoc. // OK
-* @see Some javadoc. // OK
-* @since Some javadoc. // OK
-* @serial Some javadoc. // OK
-* @serialField // OK
-* @serialData // OK
-* @deprecated Some javadoc. // OK
+* @author Some javadoc.
+* @version Some javadoc.
+* @param Some javadoc.
+* @return Some javadoc.
+* @throws Some javadoc.
+* @exception Some javadoc.
+* @see Some javadoc.
+* @since Some javadoc.
+* @serial Some javadoc.
+* @serialField
+* @serialData
+* @deprecated Some javadoc.
 */
 
+public class Example1 {}
+
 class Valid implements Serializable
 {
 }
 
 /**
 * Some javadoc.
 *
-* @since Some javadoc. // OK
-* @version Some javadoc. // Violation - wrong order
+* @since Some javadoc.
+* @version Some javadoc. // violation
 * @deprecated
-* @see Some javadoc. // Violation - wrong order
-* @author Some javadoc. // Violation - wrong order
+* @see Some javadoc. // violation
+* @author Some javadoc. // violation
```

Comment:
Please keep the reasons for violation messages, documentation examples should be as verbose as possible. You can enhance them too, if you feel they are lacking in terms of clarity.

Reply:
Instead of loose violation messages, I have added them such that they look better and don't crowd the tags
Please review if the new change is all good....